test/matchers.py

```diff
@@ -4,7 +4,7 @@
 class regex_matcher:
     """Assert that a given string meets some expectations."""
 
-    def __init__(self, pattern, flags=0):
+    def __init__(self, pattern, flags=0) -> None:
```

Comment:
are the missing types intended

Reply:
Yes, will be fixed in https://github.com/exasol/integration-test-docker-environment/issues/453